### ISSUE TITLE
A11y_Model Builder_Model Builder_Train_Usability: Keyboard focus is going to the non-interactive element "Training Completed".

### ISSUE THREAD

**Issue by Harika1702** [dotnet/machinelearning-modelbuilder#2661]
### GitHub Tags
#A11ySev4; #A11yTCS; #A11yUsable; #DesktopApp; #Win11; #Visual Studio Client; #SH_ModelBuilder_Win32_May2023; #FTP; #Win32; #Benchmark;

### Environment
Application Name: Model Builder
Microsoft Visual Studio Enterprise 2022 (64-bit) - Current
Version 17.6.1
Microsoft Windows 11 Enterprise Build 22624.1755

### Repro Steps

1. Launch the Visual Studio with at least one of the .NET workloads.
2. Tab till enable ML.NET Model Builder (Tools -> Options -> Environment -> Preview Features) and select it.
3. Create a new .NET project or open an existing .NET project.
4. Open context menu (Right-click) of the project in the solution explorer.
5. Tab till Add and then tab till Machine Learning Model and press enter.
6. Model Builder dialog will open. 
7. Complete the required details of All the three (scenario, environment, data) steps.
8. Train screen will appear.
9. Tab till "train" control and press enter. Once train done.
10. Now verify that Keyboard focus is going to the non-interactive element "Training Completed".

### Actual Result
Keyboard focus is going to the non-interactive element "Training Completed".
**Note: We are facing similar issue throughout the application.**

### Expected Result
Keyboard focus should not go to the non-interactive element "Training Completed".

### User Impact
Keyboard users will face difficulty while navigating if keyboard focus is going to the non-interactive element.

### Attachment
![Bug_Non interactive](https://github.com/dotnet/machinelearning-modelbuilder/assets/100270074/0c3294a9-bcaf-47fb-bccb-2cd7b7a5a51a)

**Comment by zewditu** [comment]
This is a contradicts the previous issue, there was an issue filed while a ago that the screen reader should  announce the status of training in live mode when the training is in progress(the time  left, failed successfully), to make that happen we should make it focusable

**Comment by zewditu** [comment]
NVM, I got the issue.

**Comment by zewditu** [comment]
@Harika1702, ready to validate by Model builder version 16.17.0.2332602.

**Comment by SachinChavhan317** [comment]
Verified the bug on below model builder version.
Version:17.17.0.2332602
The issue is fixed, hence closing the bug. #Closed;
Adding fixed attachment.
![Presentation1](https://github.com/dotnet/machinelearning-modelbuilder/assets/86241494/0ebb216b-8e05-44f6-b0c4-8553766cc251)
